templates/jobs/drupal-phpqa.yml

```diff
@@ -16,12 +16,8 @@
     - ""
   image: ghcr.io/sparkfabrik/drupal-qa:latest
   script:
-    - >
-      if [ -d ${CI_PROJECT_DIR}/src/drupal ]; then
-        ln -s ${CI_PROJECT_DIR}/src/drupal /app/drupal
-      else      
-        ln -s ${CI_PROJECT_DIR} /app/drupal
-      fi
+    - DRUPAL_ROOT_FOUND=`./bin/util finddrupal ${CI_PROJECT_DIR} ${CI_PROJECT_DIR}/src/drupal`
+    - ln -s $DRUPAL_ROOT_FOUND /app/drupal
```

Comment:
now i'm a bit confused....what's the difference to have a php script instead of the lines below ? is there any real advantage ?

Reply:
fair question, all things considered, it's only a little bit more engineered compared with the previous version; basically here we are using the script developed in https://github.com/sparkfabrik/drupal-qa/pull/2 which:

- gives us a failing error code (stopping the job) if drupal is not found in any of the dirs specified in the arguments.
 (I found out that `webflo/drupal-finder` does not recurse down in the fs tree but simply needs to receive a path and tells you, "yeah here there's a drupal or not".)
- If in the future for some reason we need to look into a third possible path where drupal can be found (beside the 2 we give here), now we only need to add a third argument, while previously we would have been force to expand the if/else/fi.

Reply:
Thanks!